index.html

```diff
@@ -229,25 +207,29 @@ <h2>
               </ul>
           </div>
         </div>
+
+        <section class="contact-me">
+          <h3 class="contact-me-header">
+           I'm always interested in hearing about new projects,
+            so if you'd like to chat please get in touch.
+          </h3>
+          <form action="https://formspree.io/f/xgedpabj" method="post" class="form-contact-me">
+            <input type="text" maxlength="30" name="username" placeholder="Full name" id="name" required>
+            <input type="email" name="email" placeholder="Full name" id="email" required>
+            <textarea name="comment" maxlength="500" id="text-area" cols="30" rows="10">Enter text here</textarea>
```

Comment:
- [ ] Kindly add `required` attribute to `textarea` tag, as the previous reviewer asked.